CONTRIBUTING.md

```diff
@@ -97,9 +97,10 @@ mainly because we use *Cypress* to run them, a traditionally end-to-end test run
 
 The logical steps in which these tests are run:
 1. You build *Elements* from the current TypeScript source.
-2. You install this *Elements* build into the chosen example project.
-3. You serve the example project on localhost over HTTP port 4200
-4. You run the *Cypress* test suite against this example application.
+2. You copy the contents of predefined example project from `examples` to `examples-dev`.
+3. You install this *Elements* build into the chosen example project.
```

Comment:
What does this line mean?

Reply:
The solution I decided to take, is that users copy the contents of `examples/angular` (or any other integration), and modify `package.json` to utilize a local build (not the ones from npm). All of this is managed by `yarn copy:angular`

Reply:
These seem like a list of instructions, steps that somebody has to take, but it is not clear what the contributor is meant to do for this step.